examples/otlp/src/main/java/io/opentelemetry/example/OtlpExporterExample.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.example;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.common.Labels;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.exporters.otlp.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+import java.util.Collections;
+
+/**
+ * Example code for setting up the OTLP exporters.
+ *
+ * <p>If you wish to use this code, you'll need to run a copy of the collector locally, on the
+ * default port. There is a docker-compose configuration for doing this in the docker subdirectory
+ * of this module.
+ */
+public class OtlpExporterExample {
+  public static void main(String[] args) throws InterruptedException {
+    // this will make sure that a proper service.name attribute is set on all the spans/metrics.
+    System.setProperty("otel.resource.attributes", "service.name=OtlpExporterExample");
+
+    // set up the span exporter and wire it into the SDK
+    OtlpGrpcSpanExporter spanExporter = OtlpGrpcSpanExporter.getDefault();
+    SimpleSpanProcessor spanProcessor = SimpleSpanProcessor.newBuilder(spanExporter).build();
```

Comment:
Are you afraid that people will copy-paste this into their production code? May be use BSP?

Reply:
🤷  There are use-cases for both kinds of span processors. If you think it's a worry, I can change it to a BSP.

Reply:
switched to BSP. we'll see if this encourages people to use it. :)